frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/Platform.java

```diff
@@ -66,14 +66,15 @@ public String getLongNodeFilename(String nodeVersion) {
 
     public String getNodeDownloadFilename(String nodeVersion) {
         if(isWindows()) {
+			int versionCompare = nodeVersion.compareTo("v4.0.0");
             if(architecture == Architecture.x64){
-                if (nodeVersion == "v4.0.0") {
+				if (versionCompare >= 0) {
```

Comment:
The point was to use the new URL mapping for Node.js versions greater than 4, so that when 4.1.0 is released, the code would still work...